framework/source/class/qx/lang/normalize/Date.js

```diff
@@ -37,6 +38,39 @@ qx.Bootstrap.define("qx.lang.normalize.Date", {
      */
     now : function() {
       return +new Date();
+    },
+
+    /**
+     * Parses a string representation of a date and return number of milliseconds since Epoch or NaN if string is unrecognised
+     *
+     * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">MDN documentation</a>
+     *
+     * Derived from <https://github.com/csnover/js-iso8601>: ©2011 Colin Snover <http://zetafleet.com>, MIT license
+     */
+    parse : function(date) {
```

Comment:
I would recommend to use `dateString` as parameter name, as it clearly tells the type and is also used at MDN "reference".
Thus making the API Doc and first lines of code like:

```
    /**
     * Parses a string representation of a date and return number of
     * milliseconds since Epoch or NaN if string is unrecognised
     *
     * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">MDN documentation</a>
     *
     * Derived from <https://github.com/csnover/js-iso8601>: ©2011 Colin Snover
     * <http://zetafleet.com>, MIT license
     *
     * @param dateString {String} A string representing an RFC2822 or ISO 8601
     *   date (other formats may be used, but results may be unexpected).
     * @return {Number|NaN} A number representing the milliseconds elapsed
     *   since January 1, 1970, 00:00:00 UTC and the date obtained by parsing
     *   the given string representation of a date.
     *   If the argument doesn't represent a valid date, NaN is returned.
     */
    parse : function (dateString)
    {
      // Match input against ISO8601 regular expression
      var captureGroups = /^(\d{4}|[+\-]\d{6})(?:-(\d{2})(?:-(\d{2}))?)?(?:T(\d{2}):(\d{2})(?::(\d{2})(?:\.(\d{3}))?)?(?:(Z)|([+\-])(\d{2})(?::(\d{2}))?)?)?$/.exec(dateString);
      if (!captureGroups) {
        return Date.originalParse(dateString);
      }
```